src/PAModel/Entropy.cs

```diff
@@ -43,6 +43,9 @@ internal class PropertyEntropy
         // Key is component name, value is Index. 
         public Dictionary<string, double> ComponentIndexes { get; set; } = new Dictionary<string, double>(StringComparer.Ordinal);
 
+        // Key is resource name, value is Index. 
+        public Dictionary<string, double> ResourceJsonIndexes { get; set; } = new Dictionary<string, double>(StringComparer.Ordinal);
```

Comment:
>double [](http://example.com/codeflow?start=34&length=6)

Curious - why are these all doubles instead of int?

Reply:
Actually.. for assets, they have names like "0001.png". Should the entropy capture the string name instead of the index. We try to regenerate a str from index, but that could be brittle (can we gaurantee server always files that format and casing?)

Reply:
let me take a look in the server to see if that's guaranteed, if so then I can get rid of this extra dictionary, since we already capture the mapping of the original names and the new names in LocalResourceFileNames dictionary

Reply:
I confirmed this only the LocalFile resources have these kind of names the Uri resources don't follow this convention.